src/graph_API.jl

```diff
@@ -0,0 +1,90 @@
+type edge
+  # param_node: the "in" node (listed first in connectparameter)
+  # var_node: the "out" node (listed second in connectparameter)
+  name::Symbol
```

Comment:
Connections can go from a var to a par with different names, so you'll need two fields here for the source and target name.